### DISCUSSION TITLE
Great Integration

### DISCUSSION THREAD

**Discussion by kwithus** [LarsK1/hass_solvis_control#14] [General]
Just want to say thanks for working on this integration. 
For sure much easier than doing yaml-modus definition .
However, as my installation is more complex (3 Heizkreise, Solar, MIA Wärmepumpe), before I can fully replace my yaml-Modbus.
So maybe, you are willing to extent the integration (I know it’s quite a bit of work)
My „wishlist“ 😉:
All 3 Heizkreise (most important)
Start/Endpoints for Zirkulation/Warmwasser/Heizkreise
Wärmepumpe

Thanks again

**Comment by LarsK1** [comment]
Hi,
danke für dein Lob. Bzgl. deine Wünsche gerne mehr Details in den jeweiligen Issues
